backend/app/environments/service.py

```diff
@@ -41,12 +41,12 @@ def create_config(
             PlatformServiceConfig.service_id == service_id,
         )
 
-        pltf_srvc_config = self.db.scalar(stmt)
-        if not pltf_srvc_config:
+        pltfrm_srvc_config = self.db.scalar(stmt)
```

Comment:
I'm not a huge fan of abbreviating these vars. Would just use `platform_service_config` in this case

Reply:
ok